cypress/integration/viewApiInformation.spec.js

```diff
@@ -1,193 +1,268 @@
+    });
 
-        it(`View API Base URLs on ${screenSize} screen`, function() {
-            cy.viewport(screenSize);
-            cy.contains(this.apiData.developmentBaseURL).should('be.visible');
-            cy.contains(this.apiData.stagingBaseURL).should('be.visible');
-        });
+    it("View application consumed by", () => {
+      cy.get(".govuk-summary-list__key").should("contain", "Manage My Home");
+    });
 
-        it(`View relevant links on ${screenSize} screen`, function() {
-            cy.viewport(screenSize);
-            var links = [
-                {text: `${this.apiData.apiName} Specification`, url: this.apiData.apiSpecificationLink},
-                {text: "GitHub Repository", url: this.apiData.githubLink},
-            ]
-
-            links.forEach(link => {
-                cy.contains(link.text).click();
-                cy.url().should("include", link.url);
-                cy.go('back');
-            });
-        });
+    it(`View relevant links on ${screenSize} screen`, function () {
+      cy.viewport(screenSize);
+      var links = [
+        {
+          text: `${this.apiData.apiName} Specification`,
+          url: this.apiData.apiSpecificationLink,
+        },
+        { text: "GitHub Repository", url: this.apiData.githubLink },
+      ];
 
-        // TODO: revise test according to what is expected. The TestAPI has 3 applications
-        // TODO: add further tests taking into consideration authorisation for editing applications
-        // it(`View applications consumed by an API on ${screenSize} screen`, function() {
-        //     cy.viewport(screenSize);
-        //     cy.contains(this.apiData.applications).should('be.visible');
-        //     cy.contains(this.apiData.application1).should('be.visible');
-        //     cy.contains(this.apiData.application2).should('be.visible');
-        //     cy.contains(this.apiData.application3).should('be.visible');
-        // });
-    })
-
-    it("Should automatically have API version selected", function() {
-        cy.get('select#VersionNo option:selected').should('have.text', this.swaggerData.info.version);
-        cy.get("a").contains("SwaggerHub").click();
-        const expectedUrl = this.swaggerData.basePath;
-        cy.url().should('include', expectedUrl);
+      links.forEach((link) => {
+        cy.contains(link.text).click();
+        cy.url().should("include", link.url);
+        cy.go("back");
+      });
     });
 
-    it("Can choose API version", function() {
-        const selectedVersion = "2.0.0";
-        cy.get('select').select(selectedVersion);
-        cy.get('select#VersionNo option:selected').should('have.text', selectedVersion);
-    });
+    // TODO: revise test according to what is expected. The TestAPI has 3 applications
+    // TODO: add further tests taking into consideration authorisation for editing applications
+    // it(`View applications consumed by an API on ${screenSize} screen`, function() {
```

Comment:
Why is this test commented out? 
We should have a test to view applications consumed by an API

Reply:
I found the test commented out when I started writing the tests then I played with it but then left it commented out. Its not needed

Reply:
I personally think this test is now needed as we need to ensure that a user is able to successfully view the applications consumed by section. Please add this test in